config/settings.yml

```diff
@@ -17,7 +17,7 @@ ui:
         - sound
       video:
   maps:
-    center: [38, -93]
+    center: [36, -77]
```

Comment:
This re-centers the map in the default map view on desktops.  The trade-off is that the map is off-center on the default map view on mobile devices.